daemon/containerd/image_manifest.go

```diff
@@ -237,3 +240,37 @@ func (m *ImageManifest) ReadConfig(ctx context.Context, outConfig interface{}) e
 
 	return readJSON(ctx, m.ContentStore(), configDesc, outConfig)
 }
+
+// PresentContentSize returns the size of the image's content that is present in the content store.
+func (m *ImageManifest) PresentContentSize(ctx context.Context) (int64, error) {
```

Comment:
Should we un-export these, given that they're only used within this package? (I think we may even be able to un-export the `ImageManifest` type)

Reply:
In this case I wanted to separate function "internal" to the `ImageManifest` itself, and "exported"  as in they are designed to be used by "outside" of the `ImageManifest` itself.

Perhaps a better solution would be to move `ImageManifest` to its own package (in `internal`).

Reply:
Ah, right, yes, that also makes a lot of sense. It's not a blocker for sure; it was mostly me starting to look "so is this used elsewhere?".


So, un-exporting the `ImageManifest` type could _slightly_ help there, but indeed ultimately moving to an `internal` package may be the best option (then we can use that approach with non-exported and exported functions to clearly indicate that separation 👍 

(definitely not for this PR though! It's fine to keep it as-is with that extra information!)